### PR TITLE
T329753 update dependencies

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator: T329753

### CODE REVIEW

File: .stylelintignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+libs/wikipedia-preview-link.css
+src/**/*.js
+src/**/*.svg
```
Comment (stephanebisson) [inline .stylelintignore:3]: We need a newline at the end of this file

File: src/init.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /* global wikipediapreview_init_options */
+/* eslint-disable camelcase */
```
Comment (stephanebisson) [inline src/init.js:2]: Could you disable only one line?

**Review by stephanebisson — CHANGES_REQUESTED** [review]

File: .stylelintignore (hunk 1 above)
Comment (medied) [reply .stylelintignore]: Done

File: src/init.js (hunk 2 above)
Comment (medied) [reply src/init.js]: Yeap

**Pull request merged**